src/java.base/share/classes/java/lang/invoke/VarHandles.java

```diff
@@ -629,6 +628,11 @@ private static void noCheckedExceptions(MethodHandle handle) {
                     noCheckedExceptions((MethodHandle) arg);
                 }
             }
+        } else {
+            /* Temporary code to handle OpenJ9's MethodHandle implementation. This will be removed in a future release. */
+            if (MethodHandles.hasCheckedException(handle)) {
```

Comment:
Updated to use the latest boolean return type approach from https://github.com/eclipse/openj9/pull/10274.